src/Slices/CompileDetails/UI/CompileStageReportTable.tsx

```diff
@@ -37,7 +37,7 @@ export const CompileStageReportTable: React.FC<Props> = ({
     <Table {...props} variant={TableVariant.compact}>
       <Thead>
         <Tr>
-          <Th key="toggle" />
+          <Th aria-hidden key="toggle"></Th>
```

Comment:
```suggestion
          <Th aria-hidden key="toggle"></Th>
```
```suggestion
          <Th aria-hidden key="toggle"/>
```

Reply:
Both are valid, but if you have any arguments about it, I'll gladly change it.

Reply:
Right, I just personally gravitate towards self-closing tags, probably as it most of the time if there are separate closing brackets then it feels like it should be something between them. But that's just me :D

Reply:
updated